README.md

```diff
@@ -101,17 +101,26 @@ pip install [--editable] .
 
 A series of tests are included for the library and the example scripts. Library tests can be found in the [tests folder](https://github.com/huggingface/transformers/tree/master/transformers/tests) and examples tests in the [examples folder](https://github.com/huggingface/transformers/tree/master/examples).
 
-These tests can be run using `pytest` (install pytest if needed with `pip install pytest`).
+These tests can be run using `unittest` or `pytest` (install pytest if needed with `pip install pytest`).
 
 Depending on which framework is installed (TensorFlow 2.0 and/or PyTorch), the irrelevant tests will be skipped. Ensure that both frameworks are installed if you want to execute all tests.
 
 You can run the tests from the root of the cloned repository with the commands:
 
+```bash
+python -m unittest discover -s transformers/tests -p "*test.py" -t .
+python -m unittest discover -s examples -p "*test.py" -t examples
+```
```

Comment:
This is much more complicated than it should be because you were relying (unknowingly, I assume) on pytest doing magic to put some directories on PYTHONPATH.

Specifically, you need the `examples/` directory to be on PYTHONPATH to run the tests there. This is what `-t examples` does in the second line.

Overall, the library could use a restructuring with:

- separate `examples`, `src` and `tests` directories
- explicit `PYTHONPATH` definitions